docs/src/app/components/pages/components/auto-complete.jsx

```diff
@@ -0,0 +1,80 @@
+const React = require('react');
+
+const { AutoComplete } = require('material-ui');
+const ComponentDoc = require('../../component-doc');
+const CodeExample = require('../../code-example/code-example');
+
+const Code = require('auto-complete-code');
+
+class AutoCompletePage extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+    return (
+      <ComponentDoc
+        name="Auto Complete"
+        componentInfo={[{
```

Comment:
I'd love to, just quite busy recently. I can do it this friday. I thought maybe we should push it first and then finish it? Since the component is usable, I wish more people can try it out before the document is ready.